eventbus/event/base.go

```diff
@@ -27,11 +27,16 @@ import (
 )
 
 // BaseMessage base of event class family, do not use it standalone
-type baseEvent struct {
+type BaseEvent struct {
+	Caller  core.RecordRef
 	Request core.RecordRef
 	Domain  core.RecordRef
 }
 
+func (be BaseEvent) GetCaller() *core.RecordRef {
```

Comment:
Rename BaseEvent into BaseLogicEvent, as I see Ledger is not planning to use this as base event. If so then rename is required.